### ISSUE TITLE
net_arch parameter setting problem

### ISSUE THREAD

**Issue by 6potato6** [cyrus-baker/RlfdQcontrol#3]
Hello, I adjusted the net_arch parameter while running a GKP state preparation experiment. I found that setting net_arch to [8,8] didn't perform as well as [1], but [8,1] performed about the same. It seems that the performance is better when the output dimension is 1. What's the reason for this? Does a more complex network result in worse performance?

**Comment by cyrus-baker** [comment]
We did observe the same phenomenon you mentioned. I believe this is simply because the relationship between parameters may be stronger as the network size increases. Therefore, you may need to more precisely tune hyperparameters such as the learning rate. For non-feedback problems, you can use the tabular method, which is similar to setting net_arch to a smaller value.

**Comment by 6potato6** [comment]
Ok. Thank you very much.
